common/src/main/java/com/github/khanshoaib3/minecraft_access/features/PlayerWarnings.java

```diff
@@ -68,48 +73,62 @@ private void loadConfigurations() {
         this.secondHealthThreshold = map.getSecondHealthThreshold();
         this.hungerThreshold = map.getHungerThreshold();
         this.airThreshold = map.getAirThreshold();
+        this.frostThreshold = map.getFrostThreshold();
     }
 
-    private void healthWarning(double health) {
+    private void healthWarning(double health, double maxHealth) {
         if (minecraftClient.player == null) return;
 
-        if (health < firstHealthThreshold && health > secondHealthThreshold && !isHealthBelowFirstThreshold && !isHealthBelowSecondThreshold) {
+        if (health <= firstHealthThreshold && health > secondHealthThreshold && !isHealthBelowFirstThreshold && !isHealthBelowSecondThreshold) {
             isHealthBelowFirstThreshold = true;
-            MainClass.speakWithNarrator(I18n.translate("minecraft_access.player_warnings.health_low", health), true);
+            MainClass.speakWithNarrator(I18n.translate("minecraft_access.player_warnings.health_low", health, maxHealth), true);
             if (playSound) minecraftClient.player.playSound(SoundEvents.BLOCK_ANVIL_LAND, 1.0f, 1.0f);
         }
 
-        if (health < secondHealthThreshold && health > 0 && isHealthBelowFirstThreshold && !isHealthBelowSecondThreshold) {
-            isHealthBelowSecondThreshold = true;
-            MainClass.speakWithNarrator(I18n.translate("minecraft_access.player_warnings.health_low", health), true);
+        if (health <= secondHealthThreshold && health > 0 && isHealthBelowFirstThreshold && !isHealthBelowSecondThreshold) {
+            isHealthBelowSecondThreshold  =true;
```

Comment:
Formatting: should be ` = `, not `  =`
```suggestion
            isHealthBelowSecondThreshold = true;
```